docs/latest/sechub-techdoc.html

```diff
@@ -12580,7 +12580,7 @@ <h4 id="section-usecase-UC_043">10.10.44. UC_043-Admin receives notification abo
 <td class="tableblock halign-left valign-top"><p class="tableblock">Inform sechub admins that new scheduler job has been started</p></td>
 <td class="tableblock halign-left valign-top"></td>
 <td class="tableblock halign-left valign-top"></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock"><em>This step is defined at method <code>notify</code> in class `com.mercedesbenz.sechub.domain.notification.superadmin.InformAdminsThatNewSchedulerInstanceHasBeenStarted`</em></p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock"><em>This step is defined at method <code>notify</code> in class `com.mercedesbenz.sechub.domain.notification.superadmin.InformAdminsThatNewSchedulerInstanceHasBeenStartedNotificationService`</em></p></td>
```

Comment:
Please replace this file content with HEAD revision.
We do not want to commit changes in latest documentation - it is auto generated and will be handled by release build.